tests/test_pump_scheduler.py

```diff
@@ -0,0 +1,47 @@
+from __future__ import absolute_import
+import unittest
+import datetime
+
+import mock
+
+from greenpithumb import pump_scheduler
+
+
+class PumpSchedulerTest(unittest.TestCase):
+
+    def setUp(self):
+        self.mock_clock = mock.Mock()
+
+    def test_wake_hour_less_than_sleep_hour(self):
+        """Test rollover from one day to the next.
+        
+        If the sleep period spans two days, the pump should not run between 
+        midnight and the wake hour.
+        """
+        self.mock_clock.now.return_value = datetime.datetime(2016, 5, 24, 0)
+        sleep_windows = ((22, 8),)
+        mock_pump_scheduler = pump_scheduler.PumpScheduler(self.mock_clock,
```

Comment:
The pump scheduler is real (it's a real instance of `PumpScheduler`), so it shouldn't have a `mock_` prefix.

The problem is that calling it `pump_scheduler` gives it the same name as the module. We can just not assign it to a variable at all and do

```python
self.assertFalse(pump_scheduler.PumpScheduler(self.mock_clock, sleep_windows).is_running_pump_allowed())
```